addons/l10n_ar/views/report_invoice.xml

```diff
@@ -185,7 +185,9 @@
                     <!-- IDENTIFICACION (ADQUIRIENTE-LOCATARIO-PRESTARIO) -->
 
                     <!-- (14) Apellido uy Nombre: Denominicacion o Razon Soclial -->
-                    <strong>Customer: </strong><span t-field="o.partner_id.commercial_partner_id.name"/>
+                    <t t-if="o.move_type == 'out_invoice'"><strong>Customer: </strong></t>
+                    <t t-else=""><strong>Supplier: </strong></t>
```

Comment:
As you are modifying a file in a localization module, could you also add the `Supplier:`  both in the pot and po file please ? This way the translation will be updated as well.